[DE] Rewrite `Client\Interface` (#6988)
ppy/osu-wiki: wiki/Client/Interface/de.md

Dies sind die Ansichtseinstellungen:

- Lokale Rangliste
- Nationale Rangliste\*
- Globale Rangliste
- Globale Rangliste (Aktive Mods)\*
- Freundesrangliste\*

\*Benötigt [osu!supporter](/wiki/osu!supporter).

Klicke auf das Sprechblasensymbol, um einen **Schnellzugang zu dieser Beatmap im Web** für die ausgewählte Beatmap zu erhalten:

- Drücke `1` oder klicke auf die Fläche `Beatmap-Übersicht/Rekorde`, um die Beatmapseite für den ausgewählten Schwierigkeitsgrad in deinem Browser zu sehen.
- Drücke `2` oder klicke auf die Fläche `Beatmap-Thema`, um die [Modding](/wiki/Modding)-Seite der Beatmap einzusehen.
- Drücke `3` oder `Esc` oder klicke auf die Fläche `Abbrechen`, um zur Songauswahl zurückzukehren.

### Beatmap-Karussell

![](img/beatmap-cards.jpg)

Das Beatmap-Karussell zeigt alle verfügbaren Beatmaps an. Unterschiedliche Beatmaps können unterschiedlich farbige Boxen haben:

| Farbe der Box | Beschreibung |
| :-: | :-- |
| **Pink** | Diese Beatmap wurde bis jetzt noch nicht gespielt. |
| **Orange** | Mindestens ein Schwierigkeitsgrad der Beatmap wurde abgeschlossen. |

> | **Hellblau** | Andere Schwierigkeitsgrade in derselben Beatmap werden angezeigt, sobald eine Beatmap erweitert wurde. |

Author: Does there really need to be a comma? If you phrase it like `Es werden andere Schwierigkeitsgrade in derselben Beatmap angezeigt, sobald eine Beatmap erweitert wurde.`, there is only one comma.

> | **Hellblau** | Andere Schwierigkeitsgrade in derselben Beatmap werden angezeigt, sobald eine Beatmap erweitert wurde. |

Reviewer: It's a weird one, cause i thought it is referring to the light blue cards being shown when selecting a beatmap. In that case it would probably have to be `wird angezeigt` anyway. :thinking: 

Can just leave it without without the comma, its fine either way.

> | **Hellblau** | Andere Schwierigkeitsgrade in derselben Beatmap werden angezeigt, sobald eine Beatmap erweitert wurde. |

Author: It indeed is refering to these blue cards seen in the picture above (beatmap-cards.jpg). I'm not sure how to express it differently.

I think I get what you mean now. Is it okay to change it to

```suggestion
| **Hellblau** | Andere Schwierigkeitsgrade in derselben Beatmap, die angezeigt werden, sobald eine Beatmap erweitert wurde. |
```

> | **Hellblau** | Andere Schwierigkeitsgrade in derselben Beatmap werden angezeigt, sobald eine Beatmap erweitert wurde. |

Reviewer: Yeah that works

Drücke `ESC` oder auf den Button `Zurück`, um zum Hauptmenü zurückzukehren.

Klicke auf den Button `Mode`, um eine Liste der verfügbaren Spielmodi in osu! anzuzeigen. Alternativ kannst du auch `Strg` und `1` (osu!), `2` (osu!taiko), `3` (osu!catch) oder `4` (osu!mania) drücken, um den Spielmodus zu ändern. Beim Wechseln zu einem anderen Modus wird auch die [Rangliste](/wiki/Ranking) ausgetauscht, was durch die Veränderung des angezeigten Rangs ersichtlich ist.

![](img/game-modifiers-DE.jpg "Spielmodifikationen")

Klicke auf den Button `Mods` oder drücke `F1`, um in die **[Modauswahl](/wiki/Game_modifier)** zu gelangen.

In diesem Bereich kannst du Modifikationen (kurz "Mods") zum Spielverlauf hinzufügen. Einige Mods verringern die Schwierigkeit und verwenden einen Multiplikator, der den von dir erreichten Punktestand verringert. Dagegen steigern manche Mods die Schwierigkeit, aber verwenden einen Multiplikator, der den von dir erreichten Punktestand erhöht. Außerdem beeinflussen manche Mods den Spielverlauf auf eine andere Weise. [Relax](/wiki/Game_modifier/Relax) und [Auto Pilot](/wiki/Game_modifier/Autopilot) fallen unter diesen Kategorie.

> Bewege deinen Cursor über das Modsymbol, um dir die Beschreibung dazu anzeigen zu lassen. Klicke auf das Modsymbol, um die Mod zu aktivieren oder zu deaktivieren. Einige Mods, wie z. B. Double Time, haben mehrere Variationen — klicke nochmals auf die Mod, um durch die Variationen durchzuwechseln. Der Punktemultiplikator zeigt den kombinierten Effekt an, die alle selektierten Mods auf deinen Punktestand haben werden. Klicke auf `Mods zurücksetzen` oder drücke `1`, um alle aktuell ausgewählten Mods zu deaktivieren. Klicke auf `Schließen`, drücke `2` oder `ESC`, um zurück zur Songauswahl zu gelangen.

Reviewer: ```suggestion
Bewege deinen Cursor über das Modsymbol, um dir die Beschreibung dazu anzeigen zu lassen. Klicke auf das Modsymbol, um die Mod zu aktivieren oder zu deaktivieren. Einige Mods, wie z. B. Double Time, haben mehrere Variationen — klicke nochmals auf die Mod, um durch die Variationen durchzuwechseln. Der Punktemultiplikator zeigt den kombinierten Effekt an, den alle selektierten Mods auf deinen Punktestand haben werden. Klicke auf `Mods zurücksetzen` oder drücke `1`, um alle aktuell ausgewählten Mods zu deaktivieren. Klicke auf `Schließen`, drücke `2` oder `ESC`, um zurück zur Songauswahl zu gelangen.
```

| [`Accuracy`](/wiki/Gameplay/Accuracy) | Wie akkurat du die Beatmap gespielt hast. Dies zählt nur, wenn der alte Punktestand überboten wird. | Ein gewichteter Durchschnitt der Genauigkeit deiner besten Scores. |
| `Max Combo` | Die höchste Combo, die du auf der Beatmap gespielt hast. | Die höchste Combo, die du jemals über alle Beatmaps gesehen erreicht hast. |
| [`Ranked Score`](/wiki/Gameplay/Score/Ranked_score) | Dein [bestes Ergebnis](/wiki/Gameplay/Score/Ranked_score) auf der Beatmap. | Die Anzahl der Punkte von allen gerankten Beatmaps, die du jemals gespielt hast, wobei jede Beatmap genau einmal gezählt wird. |
| [`Total Score`](/wiki/Gameplay/Score/Total_score) | Das wird nicht berücksichtigt, da es deine Position in der Online-Rangliste nicht beeinflusst. | Das gleiche wie beim gerankten Score, aber es werden alle verfügbaren Beatmaps auf der osu!-Webseite miteinbezogen, also auch gescheiterte Beatmaps oder Beatmaps, bei denen der alte Punktestand nicht überboten wurde. Das wird auf dein [Level](/wiki/Gameplay/Score/Total_score#level) angerechnet.<!-- TODO: Check link as soon as article is available --> |

> | [`Performance`](/wiki/Performance_points) | Die Menge an [ungewichteten PP](/wiki/Performance_points#why-didnt-i-gain-the-full-amount-of-pp-from-a-map-i-played?), die du für das Spiel erreichen würdest. | Deine gesamte Menge an Performance-Punkten und wie viele PP das eingereichte Spiel wert war.<!-- TODO: Check link as soon as article is available --> |

Reviewer: ```suggestion
| [`Performance`](/wiki/Performance_points) | Die Menge an [ungewichteten PP](/wiki/Performance_points#why-didnt-i-gain-the-full-amount-of-pp-from-a-map-i-played?), die du für den Score erreichen würdest. | Deine gesamte Menge an Performance-Punkten und wie viele PP der eingereichte Score wert war.<!-- TODO: Check link as soon as article is available --> |
```
I think this would be more understandable than `Spiel`.
